VCFv4.3.tex

```diff
@@ -292,16 +292,16 @@ \subsection{Header line syntax}
 
 \subsection{Data lines}
 All data lines are tab-delimited
-with no tab character at the end of the line. The last data line should end with a line separator. In all cases,
+with no tab character at the end of the line. The last data line must end with a line separator. In all cases,
 missing values are specified with a dot (`.').
 
 \subsubsection{Fixed fields}
 There are 8 fixed fields per record.  Fixed fields are:
 
 \begin{enumerate}
-  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM should form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
+  \item CHROM - chromosome: An identifier from the reference genome or an angle-bracketed ID String (``$<$ID$>$'') pointing to a contig in the assembly file (cf. the \#\#assembly line in the header). All entries for a specific CHROM must form a contiguous block within the VCF file. The colon symbol (:) must be absent from all chromosome names to avoid parsing errors when dealing with breakends. (String, no white-space permitted, Required).
```

Comment:
We could always *add* something to a VCF that would indicate that is is sorted according to the original reference, similar to the sort order header tag in a SAM file.  That would not invalidate previous VCFs.  In many, many tools I write, I do not need the reference explicitly, even when merging VCFs.